<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelWycen</class>
 <widget class="QWidget" name="PanelWycen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>481</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Id Wyceny</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ID_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Opis Eksperta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Opis_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cena</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cena_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="Usun_Button">
         <property name="text">
          <string>Usun Wycene</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Edytuj_Button">
         <property name="text">
          <string>Edytuj Wycene</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Wroc_Button">
         <property name="text">
          <string>Wroc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tablica"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
